### PR TITLE
Add netsetup=dhcp documentation

### PR DESCRIPTION
**Pull request by ereslibre**

### CODE REVIEW

File: app/views/setup/worker_bootstrap.html.slim [hunk 1]
```diff
@@ -11,15 +11,16 @@ h1 Bootstrap your Container as a Service Platform
...
-  |  The automatic installation gets invoked by adding autoyast=#{autoyast_url} to the kernel parameter list.
-  |  As installation media, you can use the very same image you bootstrapped the admin node with.
+  | The automatic installation gets invoked by adding autoyast=#{autoyast_url} to the kernel parameter list.
+  | If you aren't under a PXE environment you can also use netsetup=dhcp kernel parameter for the network to be automatically configured using a reachable DHCP server.
+  | As installation media, you can use the very same image you bootstrapped the admin node with.
 
-  |  A ready to use AutoYaST profile has been already generated for you during the bootstrap of the admin node.
-  |  Bootstrap all the nodes you want to make part of this platform by adding the following boot parameter
-  |  autoyast=#{autoyast_url}
+  | A ready to use AutoYaST profile has been already generated for you during the bootstrap of the admin node.
```
Comment (monstermunchkin) [inline app/views/setup/worker_bootstrap.html.slim]: … has already been …

Comment (ereslibre) [inline app/views/setup/worker_bootstrap.html.slim]: Addressed.

File: app/views/setup/worker_bootstrap.html.slim [hunk 2]
```diff
@@ -11,15 +11,16 @@ h1 Bootstrap your Container as a Service Platform
 
 p
   | In order to complete the installation, it is necessary to bootstrap a few additional nodes, those will be the Kubernetes Master and Workers.
-  |  This process leverages AutoYaST and is (almost) fully automated.
-  |  In case you are not familiar with it, you can find more information about AutoYaST in the&nbsp;
+  | This process leverages AutoYaST and is (almost) fully automated.
+  | In case you are not familiar with it, you can find more information about AutoYaST in the&nbsp;
```
Comment (jimmykarily) [inline app/views/setup/worker_bootstrap.html.slim]: I think now the resulting text won't have a space after the "."

Comment (ereslibre) [inline app/views/setup/worker_bootstrap.html.slim]: Right, fixed it directly on `master`, since it was blank space fixing.

**Pull request merged**